src/BloomBrowserUI/karma.conf.js

```diff
@@ -89,7 +89,7 @@ module.exports = function (config) {
         // - Safari (only Mac)
         // - PhantomJS
         // - IE (only Windows)
-        browsers: ['Firefox'],
+        browsers: ['Chrome'],
```

Comment:
Oops! I goofed here. When I said the Overflow tests passed I wasn't on this branch, so ran Firefox, of course. The Overflow tests are set to not run at all on Chrome. We need to modify this line to show "browsers: ['Firefox', 'Chrome']," That way, the Firefox will run the Overflow tests and Chrome will debug.